WooCommerce/Classes/ViewRelated/Products/ProductLoaderViewController.swift

```diff
@@ -169,19 +154,29 @@ private extension ProductLoaderViewController {
         }
     }
 
-    /// Presents the ProductDetailsViewController, as a childViewController, for a given Product.
+    /// Presents the ProductDetailsViewController or the ProductFormViewController, as a childViewController, for a given Product.
     ///
     func presentProductDetails(for product: Product) {
-        let detailsViewModel = ProductDetailsViewModel(product: product, currency: currency)
-        let detailsViewController = ProductDetailsViewController(viewModel: detailsViewModel)
+
+        let isFeatureFlagOn = ServiceLocator.featureFlagService.isFeatureFlagEnabled(.editProducts)
+        let viewController: UIViewController
+        if product.productType == .simple && isFeatureFlagOn {
+            viewController = ProductFormViewController(product: product, currency: currency)
+            // Since the edit Product UI could hold local changes, disables the bottom bar (tab bar) to simplify app states.
```

Comment:
nit: we probably don't need this comment anymore since it's embedded in a modal?

Reply:
Oops, fixed! 👍